src/app/services/auth/auth.guard.spec.ts

```diff
@@ -5,6 +5,7 @@ import {HttpClient} from '@angular/common/http';
 import { StoreModule } from '@ngrx/store';
 import { of } from 'rxjs';
 import {AuthGuard} from './auth.guard';
```

Comment:
Can I be awkward and ask you to clean up the imports up here? This one looks out of place now. :)